airbyte-persistence/job-persistence/src/test/java/io/airbyte/persistence/job/DefaultJobPersistenceTest.java

```diff
@@ -136,7 +136,10 @@ static void dbDown() {
     container.close();
```

Comment:
My IntelliJ autoformatter had a lot of opinions about this file, I will annotate where the actual changes are